src/Microsoft.AspNetCore.Blazor.Browser.JS/src/Services/UriHelper.ts

```diff
@@ -22,7 +22,11 @@ registerFunction(`${registeredFunctionPrefix}.enableNavigationInterception`, ()
     const anchorTarget = findClosestAncestor(event.target as Element | null, 'A');
     if (anchorTarget) {
       const href = anchorTarget.getAttribute('href');
-      if (isWithinBaseUriSpace(toAbsoluteUri(href))) {
+      //if the user wants to user some specific browser/OS feature, we dont handle it and let the browser/OS 
+      var anyChangeBehaviorKeyHold = event.ctrlKey || event.shiftKey || event.altKey || event.metaKey;
+      if (isWithinBaseUriSpace(toAbsoluteUri(href)) && 
+          !anyChangeBehaviorKeyHold ) 
+      {
```

Comment:
Really minor comments, but could you use `const` instead of `var` on line 26? Also lines 27 and 28 can easily go on the same line as they are short now.